packages/compass-e2e-tests/tests/instance-sidebar.test.ts

```diff
@@ -154,9 +154,16 @@ describe('Instance sidebar', function () {
     }
 
     // open the create database modal from the sidebar
-    await browser.clickVisible(Sidebar.CreateDatabaseButton, {
-      screenshot: 'before-can-create-a-database-and-drop-it-click.png',
-    });
+    if (TEST_MULTIPLE_CONNECTIONS) {
```

Comment:
These are drive-bys: I realised we weren't properly scoping the selector to the relevant connection.